package.json

```diff
@@ -1,8 +1,9 @@
 {
   "license": "MIT",
   "devDependencies": {
-    "parcel-bundler": "^1.12.5",
-    "purescript": "^0.14.3",
+    "parcel": "^2.6.0",
+    "process": "^0.11.10",
```

Comment:
Could you clarify why `process` is being added here?

Reply:
Good catch, since I can't recall I must have added it by accident.

Reply:
Okay, I think I found it.

It will be added automatically as part running `make testAllCI`. `SignalRenderJs` is using process. parcel figures out and magically auto installs it.


![image](https://user-images.githubusercontent.com/13085980/178559515-9c2530ac-5a66-4e87-844f-2a3ae8a17029.png)

https://parceljs.org/features/node-emulation/#polyfilling-%26-excluding-builtin-node-modules

![image](https://user-images.githubusercontent.com/13085980/178560341-f8e7940f-ab45-4f33-ae2f-b6ed2ce5fe56.png)

Should we add it back in or let parcel add it (creating a diff) for the user in this case?

Reply:
Ok, thanks for clarifying. Let's just keep `process` as a dependency then.

Reply:
Added back in